lib/KalturaHelpers.php

```diff
@@ -200,8 +200,8 @@ public static function getOption( $name, $default = null ) {
 		$name    = is_string( $name ) ? $name : null;
 		$default = is_bool( $default ) ? $default : null;
 
-		$value = get_option( $name, $default );
-		if ( ! is_null( $value ) ) {
+		$value = self::isPluginNetworkActivated() ? get_site_option( $name, $default ) : get_option( $name, $default );
```

Comment:
But there are 2 options:
1. The plugin can be "Network activated", which means that the network admin configures the Kaltura configuration and these settings are applied to all sites.
2. Each site has its own configuration.

Now our plugin supports both cases. I tested it and it worked fine.